scipioninstaller/installer.py

```diff
@@ -256,11 +271,16 @@ def main():
         checkProgram(GIT) if dev else None
         # Check Scipion home folder and create it if apply.
         solveScipionHome(scipionHome, dry, noAsk)
-        cmd = getEnvironmentCreationCmd(conda, scipionHome)
+        try:
+            scipionEnv = str(args.scipionEnv)
+        except:
+            scipionEnv = SCIPION_ENV
```

Comment:
Be careful, args.scipionEnv will be `None` if not passed, then `str(None)` would be `'None'` instead of `SCIPION_ENV`.

Reply:
SCIPION_ENV = '.scipion3env' by default.

Reply:
You have reason David....thanks....

Reply:
I'd use default in the argument definition, and these lines couuld go away if you use args.scipionEnv, or then replace these lines with just the assignment.

https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument